docs/reference/numerical-types.md

```diff
@@ -104,18 +104,19 @@ printf("Index %" PRI_PlatformIndexType ". Min %" PRI_PlatformIndexType, index, s
 ```
 
 > [!NOTE]
-> in order for F´ to compile without warnings it is necessary that each of the platform types are elements in the set of integers supplied by the C standard integers header (`stdint.h`). i.e. each type must be an `int8_t`, `int16_t`, `int32_t`, `int64_t` or unsigned variants. On some compilers `int` and `unsigned int` are not members of that set and on those platforms it is imperative that both `PlatformIntType` and `PlatformUIntType` be set to some fixed size type instead.
+> in order for F´ to compile without warnings it is necessary that each of the platform types are elements in the set of integers supplied by the C standard integers header (`stdint.h`). i.e. each type must be an `int8_t`, `int16_t`, `int32_t`, `int64_t` or unsigned variants. On some compilers `int` and `unsigned int` are not members of that set and on those platforms `PlatformIntType` and `PlatformUIntType` should not be serialized.
 
 ### Configurable Integer Types
 
 Project may configure the framework types that the framework and components use for implementation through
 `FpConfig.h`. The default configuration as supplied with F´ uses the above platform types where applicable.
 
-| Framework Type  | Logical Usage        | Default               | Format Specifier    | Notes |
-|-----------------|----------------------|-----------------------|---------------------|-------|
-| FwIndexType     | Port indices         | PlatformIndexType     | PRI_FwIndexType     |       |
-| FwSizeType      | Sizes                | PlatformSizeType      | PRI_FwSizeType      |       |
-| FwAssertArgType | Arguments to asserts | PlatformAssertArgType | PRI_FwAssertArgType |       |
+| Framework Type   | Logical Usage        | Default                | Format Specifier     | Signed | Size            |
+|------------------|----------------------|------------------------|----------------------|--------|-----------------|
+| FwIndexType      | Port indices         | PlatformIndexType      | PRI_FwIndexType      | Yes    | Minimum 1 Byte  |
```

Comment:
We had a conversation about using `FwIndexType` for broader use cases, such as array indices (as long as it's not an enormous array), loops, etc.
Should it be called out here?